core/src/Kokkos_MemoryPool.hpp

```diff
@@ -461,7 +465,7 @@ class MemoryPool {
    */
   KOKKOS_FORCEINLINE_FUNCTION
   uint32_t get_block_size_lg2(uint32_t n) const noexcept {
-    const unsigned i = Kokkos::Impl::integral_power_of_two_that_contains(n);
+    const unsigned i = integral_power_of_two_that_contains(n);
```

Comment:
I haven't looked too carefully how performance critical that function might be.
If necessary we could prefer the `*_builtin` overloads in `Experimental::`